sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/AbstractiveSummaryResult.java

```diff
@@ -43,6 +50,16 @@ public AbstractiveSummaryResult(String id, TextDocumentStatistics textDocumentSt
         super(id, textDocumentStatistics, error);
     }
 
+    /**
+     * Get the detectedLanguage property: If 'language' is set to 'auto' for the document in the request this field will
+     * contain a 2 letter ISO 639-1 representation of the language detected for this document.
```

Comment:
I know this docstring came from the swagger, but I'm updating to say that it returns the DetectedLanguage object. It's kind of misleading right now saying it's going to return the 2 letter string.